profiles/common/modules/custom/multisite_config/lib/Drupal/locale/Config.php

```diff
@@ -61,8 +61,12 @@ public function setLanguageNegotiation($language_negotiation, $type = LANGUAGE_T
    *    Language prefix.
    */
   public function setLanguagePrefix($language, $prefix) {
-    $arguments = array(':language' => $language, ':prefix' => $prefix);
-    db_query("UPDATE {languages} SET prefix = ':prefix' WHERE language = ':language'", $arguments)->execute();
+    db_update('languages')
+      ->fields(array(
+        'prefix' => $prefix,
+      ))
+      ->condition('language', $language, '=')
```

Comment:
imo, the condition is more clear without the operator

> If no operator is specified, = is assumed. That means the most common case would be condition('myfield', $value), which results in a conditional fragment of myfield = :value, where :value will be replaced with $value when the query is run.

If you agree i will make the small change